src/gui/creds/oauth.cpp

```diff
@@ -110,7 +119,9 @@ void OAuth::start()
                         || jsonData.isEmpty() || json.isEmpty() || refreshToken.isEmpty() || accessToken.isEmpty()
                         || json["token_type"].toString() != QLatin1String("Bearer")) {
                         QString errorReason;
-                        QString errorFromJson = json["error"].toString();
+                        QString errorFromJson = json["error_description"].toString();
+                        if (errorFromJson.isEmpty())
+                            QString errorFromJson = json["error"].toString();
```

Comment:
BUG! two times "QStrings errorFromJson" in different scope.

Reply:
well spotted.